gitCore/jGit/src/main/java/com/virtuslab/gitcore/impl/jgit/GitCoreCommit.java

```diff
@@ -22,7 +24,7 @@ public GitCoreCommit(@NonLeaked RevCommit commit) {
     this.shortMessage = commit.getShortMessage();
     this.commitTime = Instant.ofEpochSecond(commit.getCommitTime());
     this.hash = GitCoreCommitHash.of(commit.getId());
-    this.treeHash = GitCoreTreeHash.of(commit.getTree().getId());
```

Comment:
Huh I'd leave as is, `.of()` looks confusing as hell

Reply:
Although `commit.getTree().getId().toGitCoreTreeHash()` would be quite reasonable